src/Uno.UI.Runtime.Skia.Gtk/GtkCoreWindowExtension.cs

```diff
@@ -98,7 +98,12 @@ private void OnWindowLeaveEvent(object o, LeaveNotifyEventArgs args)
 			{
 				if (AsPointerArgs(args.Event) is { } ptArgs)
 				{
-					_ownerEvents.RaisePointerExited(ptArgs);
+					// The Ungrab mode event is triggered after click 
+					// even when the pointer does not leave the window
+					if (args.Event.Mode != CrossingMode.Ungrab)
```

Comment:
We could have put this before the `AsPointerArgs(args.Event)`, it would avoid useless work :)

Reply:
Also the "grab" refers to the "pointer capture" which is not (really) supported yet on GTK. We might have to review this when we add native support of capture on GTK.

Reply:
I will note this in the comment 👍